apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -143,27 +136,31 @@ function NotificationList() {
         }
       />
       <TemplateListTableWrapper>
-        <Table
-          onRowClick={onRowClick}
-          loading={isLoading || areNotificationGroupLoading}
-          data-test-id="notifications-template"
-          columns={columns}
-          data={templates || []}
-          pagination={{
-            pageSize: pageSize,
-            current: page,
-            total: totalTemplatesCount,
-            onPageChange: handleTableChange,
-          }}
-          noDataPlaceholder={
-            <TemplatesListNoData
-              onCreateClick={() => handleRedirectToCreateTemplate(false)}
-              onTryDigestClick={handleCreateDigestDemoWorkflow}
-              tryDigestDisabled={isTryDigestDisabled}
-            />
-          }
-        />
-        {/* <TemplatesStoreModal /> */}
+        {hasTemplates ? (
+          <Table
+            onRowClick={onRowClick}
+            loading={isLoading || areNotificationGroupLoading}
+            data-test-id="notifications-template"
+            columns={columns}
+            data={templates}
+            pagination={{
+              pageSize: pageSize,
+              current: page,
+              total: totalTemplatesCount,
+              onPageChange: handleTableChange,
+            }}
+          />
+        ) : (
+          <TemplatesListNoData
+            blueprints={popularBlueprints}
+            isLoading={areBlueprintsLoading}
+            allTemplatesDisabled={areBlueprintsLoading || !hasGroups}
```

Comment:
When the blueprints are loading or the response doesn't have groups - the `All Templates` tile will be disabled. It should never happen for the Novu Cloud but might be the case for the forks/on-prem.